tlatools/org.lamport.tlatools/src/tla2sany/xml/XMLExporter.java

```diff
@@ -43,14 +43,31 @@
 
 public class XMLExporter {
 
+  /**
+   * CLI help text listing various options.
+   */
+  private static final String HELP_TEXT =
+    "SANY XML Exporter\n" +
+    "Usage:\n\n" +
+    "  java tla2sany.xml.XMLExporter [-o|-t] [-I <dir_path>]* <input_file>.tla\n\n" +
```

Comment:
Assuming the * means "0 or more instances of -I dir_path", do you have other examples of similar usages in man pages / helps?

Reply:
Switched to using the UsageGenerator; see initial comment for example of help text generated.